### DISCUSSION TITLE
[Nodejs]: axios throws "invalid url" errors

### DISCUSSION THREAD

**Discussion by aladdin-add** [apache/skywalking#9642] [Q&A] [answered]
we were using the node.js sdk `skywalking-backend-js@v0.5.1` with `axios@0.27.2`. but it throws an error like `invalid url`.

the repro code is really simple:

```js
const axios = require('axios');
const ax = axios.create({baseURL: 'https://x.d.so/'});

const res = await ax.get('/xxxx'); // throws here.😪
```

I noticed it has a TODO in the codebase: https://github.com/apache/skywalking-nodejs/blob/17bc0bf2015eed84de57c095d12f0598df11dc90/src/plugins/AxiosPlugin.ts#L46. is there a plan to fix it? 🙌

**Comment by kezhenxu94** [comment, marked as answer]
Fixed in https://github.com/apache/skywalking-nodejs/pull/93

Can you take a look? @aladdin-add
